model/llm/prompt/parse.go

```diff
@@ -17,28 +17,39 @@ var (
 func ParseResponse(response string) (assessment metrics.Assessments, code string) {
 	assessment = metrics.Assessments{}
 
+	if isOnlyWhitespace(response) {
+		return assessment, response
+	}
+	assessment[metrics.AssessmentKeyResponseNotEmpty] = 1
```

Comment:
Please always use "+=" for all metrics. If you set it to "0" same thing `+= 0` we are adding to the assessment. If there are multiple additions, we then don't have to handle setting them, because they are just added.